src/lib/core/coio.h

```diff
@@ -169,7 +169,7 @@ enum {
 /**
  * Wait until READ or WRITE event on socket (\a fd). Yields.
  * \param fd - non-blocking socket file description
- * \param events - requested events to wait.
+ * \param event - requested events to wait.
  * Combination of TNT_IO_READ | TNT_IO_WRITE bit flags.
  * \param timeout - timeout in seconds.
  * \retval 0 - timeout
```

Comment:
It's a bit too picky, but on the 'just asked' right: shouldn't we strip the dashes?

![image](https://user-images.githubusercontent.com/92749/227658809-24dfb5e8-3c02-4ab5-8bac-7b0a3d1fffd2.png)

Feel free to ignore.

Reply:
removed dashes